versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/logic/RepositoryDescription.java

```diff
@@ -72,6 +75,16 @@ interface Builder {
 
   String defaultBranchName();
 
+  /**
+   * The timestamp when the repository has been fully imported. This information is only present if
+   * the repository was imported using a recent version of Nessie CLI.
+   */
+  @JsonSerialize(using = InstantSerializer.class)
+  @JsonDeserialize(using = InstantDeserializer.class)
+  @Nullable
+  @jakarta.annotation.Nullable
+  Instant repositoryImportedTime();
```

Comment:
Please add an entry to CHANGELOG saying that the version of the Nessie service must be at least 0.68.0, if the repo was imported with 0.68.0 or newer.